pcsx2/gui/FrameForGS.cpp

```diff
@@ -557,6 +557,7 @@ void GSFrame::OnUpdateTitle( wxTimerEvent& evt )
 #endif
 
 	double fps = wxGetApp().FpsManager.GetFramerate();
+	float per = gsRegionMode == Region_NTSC ? (fps * 100) / EmuConfig.GS.FramerateNTSC.GetWhole() : (fps * 100) / EmuConfig.GS.FrameratePAL.GetWhole(); 
```

Comment:
I would just display an integer percent. I think this values is refreshed once per second? If it is refreshed once per second and you just count vsyncs your precision is aproximately as high as 1/50 or 1/60. Accordingly 2% should be something like minimal increment and therefore 1% sufficient for display. If you really mean the timespan between (several) vsyncs you can have higher precision under perfect conditions. Still 1% will be sufficient.